src/label_studio_sdk/_extensions/label_studio_tools/core/utils/json_schema.py

```diff
@@ -65,6 +68,7 @@ def json_schema_to_pydantic(json_schema: dict, class_name: str = 'MyModel') -> G
     json_schema_str = json.dumps(json_schema)
     
     # Generate Pydantic model code from the JSON schema string
+    logger.debug(f"Generating Pydantic model code from json schema: {json_schema_str}")
```

Comment:
remove debug log? or will this never be enabled in prod?

Reply:
yes, for not production use, these logs are helpful to debug structured output schema